java-checks/src/test/files/checks/NoTestInTestClassCheck.java

```diff
@@ -87,7 +88,13 @@ public void test2() {
 public class MyNewTest { // should not raise an issue
 }
 @org.junit.runner.RunWith(Enclosed.class)
-public class MyNewTest2 { // no issue
+public class MyNew2Test { // no issue
+}
+@org.junit.runner.RunWith(cucumber.api.junit.Cucumber.class)
+public class MyCucumberTest { // should not raise an issue
+}
+@org.junit.runner.RunWith(Cucumber.class)
```

Comment:
Indeed @vilchik-elena we are currently specifically targeting `Cucumber.class` and `Enclosed.class`